### PR TITLE
4 create database migrations

### LINKED ISSUE #4: Create Database Migrations

### PR DESCRIPTION
**Pull request by SirrorsMoore1975**
Resolves #4

### REQUIRED:
- [x] Write the issue number associated with this pull request after "Resolves #"
- [x] Add the correct label for this pull request (you can just use the same label as the associated issue)
- [x] Be sure to share a link to this pull request in Slack/Discord and ask for a review! <3

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+DB_NAME=
+DB_USER=
+DB_PASSWORD=
```
Comment (kappanjoe) [inline .env.example:3]: If we can set up different build environments (see comment on `config/settings.py`), let's add the `DATABASE_URL` variable here as well, please.

**Review by kappanjoe — CHANGES_REQUESTED** [review]
Just some build environment fixes as noted in the comments. Additionally, are these migrations set up to run upon build/deployment?

File: base/models.py [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
 # Create your models here.
+
+class Location(models.Model):
+    country = models.CharField(max_length=255, unique=True)
+    prefecture = models.CharField(max_length=255)
+
+class Store(models.Model):
+    name = models.CharField(max_length=255)
+    location_id = models.ForeignKey(Location, on_delete=models.CASCADE)
+    coordinates = models.PointField(srid=4326) # ID for goggle 4326 PointField(srid=4326)
```
[the remark this answers is not in the document]
Comment (SirrorsMoore1975) [reply base/models.py:12]: According to Ikuno, I was told using srid=4326 should give you coordinates in the form of Latitude and Attitude, please check if this is correct.

**Review by kappanjoe — APPROVED** [review]

**Comment by kappanjoe** [comment]
LGTM!

**Pull request merged**